src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/TraceLogging/EnumerableTypeInfo.cs

```diff
@@ -38,7 +38,7 @@ public EnumerableTypeInfo(Type type, TraceLoggingTypeInfo elementInfo)
 
         public override void WriteData(TraceLoggingDataCollector collector, PropertyValue value)
         {
-            int bookmark = collector.BeginBufferedArray();
+            int bookmark = TraceLoggingDataCollector.BeginBufferedArray();
```

Comment:
Is the `collector` argument ever used by any of the `WriteData` overrides now?

Reply:
Removed